modules/apps/portal-search/portal-search-rest-test/src/testIntegration/java/com/liferay/portal/search/rest/resource/v1_0/test/SearchResultResourceTest.java

```diff
@@ -1394,6 +1434,7 @@ null, null, _getSearchFacets(jsonObject),
 	private JSONFactory _jsonFactory;
 
 	private Locale _locale;
+	private ObjectDefinition _objectDefinition;
```

Comment:
@vngoliveira can you try to use the `@DeleteAfterTestRun` annotation instead of using the `tearDown` method?